### ISSUE TITLE
Can I set more than one Base_Url?

### ISSUE THREAD

**Issue by foxinstone** [DanielnetoDotCom/YouPHPTube#1080]
When I put YouPhpTube on an intranet server and connect to an external  VPS using VPN, and then use Nginx reverse proxy access it. There are three access situations:
(1)192.168.1.1(this is the YPT server IP);
(2)192.168.50.3(this is the YPT server IP on VPN)
(3)https://xxxxxx.com/(this is domain name)
I have tested it, and one of these three situations can work as long as they are properly configured.
However, I actually want to work in all three situations(Because each of the three situations has its own use.), can I?

**Comment by DanielnetoDotCom** [comment]
Sorry I cant help you with Nignx reverse proxy

**Comment by nightstryke** [comment]
@foxinstone You're going to have to bite the bullet on this one, YouPHPTube is not setup for Reverse Proxy. Quite honestly even Nginx is a little sketchy when trying to use it with YouPHPTube. You should just use the latest version of Apache and install Nginx as the RTMP Server. I've tried the other methods they don't work well if at all.

**Comment by foxinstone** [comment]
This is not a question about Nginx reverse proxy, which is about rewriting mechanisms.
In fact, I used Nginx reverse proxy, which worked fine. This is one of the three situations I said.
We may be able to learn from other software experiences, such as NextCloud, which allows to leave the rewrite url blank, so that the URL entered by the user will not be changed.
The question I asked is actually similar to issues 1008. You may be too busy to see it clearly.

**Comment by DanielnetoDotCom** [comment]
Sorry, that is not the default behavior.

it was never tested, but I guess you will need to change the configuration table on encoder database.

try to make the allowedStreamersURL column blank.

**Comment by foxinstone** [comment]
Thank you for your reply!
Yes, I have checked the encoder database to make sure the value of the allowedStreamersURL column is blank.
According to the web page response, it is not that the streamer site cannot be found or allowed, but the username/password is incorrect.

**Comment by DanielnetoDotCom** [comment]
I see, yeah, I just realize you need a full domain to an YouPHPTube  site on the Streamer Site URL

**Comment by foxinstone** [comment]
In order to avoid the problem of Nginx reverse proxy, can we consider the problem of installing a site on different ports on the LAN.
Is it not possible to build these two stations with IP addresses only on the LAN?

**Comment by foxinstone** [comment]
But the current problem happens to be nothing to Nginx. There is a problem with the interconnection of the two sites.

**Comment by foxinstone** [comment]
Simply put, it seems to be the case: when the streamer's baseURL = '/', and when the two sites are installed on different ports, there is a problem with the interconnect. (Use IP address, not related to Nginx)

**Comment by DanielnetoDotCom** [comment]
you cant connect on the encoder using / as a streamer site.

the URL must be a real streamer.

The encoder authenticate on the Streamer site, so it needs to know where is your location

**Comment by foxinstone** [comment]
Ok, don't bother you much.
I made the final effort in this way: 
If the streamer baseURL is set to '/', the streamer can be accessed normally in three ways, which means that installing YPT on the internal server is also perfect, which is very good for the media server. Important because the intranet server can achieve high performance inexpensively. I didn't expect the result to be such a surprise, because I saw Nginx's pessimistic tone...
So, if you and your colleagues can also notice this, just improve the interconnection with the Encoder at streamer‘s baseURL='/' (using IP and port, without reverse proxy), isn't that perfect? ?
I sincerely hope that this software is getting better and better!
Thank you!
